docs/README.md

```diff
@@ -2568,7 +2568,7 @@ HTTPie plays exceptionally well with the following tools:
 
```

Comment:
also while at it you can update the JQ site to new home: `https://jqlang.github.io/jq/`